### PR TITLE
Added in an assertion for test_full_smrf

### LINKED ISSUE #128: test_variables in test_full_smrf.py doesn't assert anything
[Line 44](https://github.com/USDA-ARS-NWRC/smrf/blob/e4d0f67ba2bdf32c44d383b6ffe10657020d497d/tests/test_full_smrf.py#L44) doesnt't assert anything. And then in [Line 81](https://github.com/USDA-ARS-NWRC/smrf/blob/e4d0f67ba2bdf32c44d383b6ffe10657020d497d/tests/test_full_smrf.py#L81) also doesn't assert anything so the test is always successful. GULP.

### PR DESCRIPTION
**Pull request by micahjohnson150**
Added in an assertion for test_full_smrf on the variables gold checking. Fixed #128, Also autopep8 and isort plus some light clean up.

### CODE REVIEW

File: tests/test_full_smrf.py [hunk 1]
```diff
@@ -8,28 +8,30 @@
...
+
 import numpy as np
 from netCDF4 import Dataset
 
-def compare_image(v_name, gold_dir, test_dir, msg):
+import smrf
+from smrf.framework.model_framework import run_smrf
+
+
+def compare_image(v_name, gold_dir, test_dir):
```
Comment (scotthavens) [inline tests/test_full_smrf.py]: Is this used anywhere else?

Comment (micahjohnson150) [reply tests/test_full_smrf.py]: run_smrf or compare_image?

run _smrf is used in the setUpClass function to build the data
compare_image is used in the loop to test all the variables.

Comment (scotthavens) [reply tests/test_full_smrf.py]: `compare_image`

File: tests/test_full_smrf.py [hunk 2]
```diff
@@ -57,28 +59,28 @@ def setUpClass(self):
...
-                     'precip', 'thermal', 'wind_speed','wind_direction',
-                     'snow_density', 'vapor_pressure']
+        variables = ['air_temp', 'precip_temp', 'net_solar',
+                     'percent_snow', 'precip', 'thermal', 'wind_speed',
+                     'wind_direction', 'snow_density', 'vapor_pressure']
 
         for v in variables:
-            msg = "{} does not match the gold file".format(v)
-            compare_image(v, self.gold, self.output, msg)
+            self.assertTrue(compare_image(v, self.gold, self.output))
```
Comment (micahjohnson150) [inline tests/test_full_smrf.py]: @scotthavens compare_image gets used here.

File: tests/test_full_smrf.py (hunk 1 above)
Comment (micahjohnson150) [reply tests/test_full_smrf.py]: compare_image is only used in test_full_smrf, so if someone does use it anywhere else they will need to assertTrue on it.

**Comment by micahjohnson150** [comment]
@robertson-mark  click the approve ....

**Pull request merged**